e2e-tests/playwright/e2e/github-happy-path.spec.ts

```diff
@@ -89,14 +100,80 @@ test.describe.skip('GitHub Happy path', () => {
     await expect(page.locator(`text=${issuesCountText}`)).toBeVisible();
 
     for (const issue of openIssues.slice(0, 5)) {
-      const issueLocator = page.locator(
-        `text=${issue.title.replace(/\s+/g, ' ')}`,
+      const issueSelector = `text=${issue.title.replace(/\s+/g, ' ')}`;
```

Comment:
can you create `(/\s+/g` as a constant and give a name which explains the usage?

Reply:
It's a simple, self-explanatory regex. It is replacing two or more spaces with a single space.